src/themes/teams/components/Icon/svg/icons/add.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default (
+  <svg viewBox="0 0 32 32" role="presentation" className="app-svg icons-add" focusable="false">
+    <g className="icons-default-fill">
+      <path
+        className="icons-unfilled"
+        d="M23.3516,16.1172C23.4502,16.2168,23.5,16.334,23.5,16.4688c0,0.1357-0.0498,0.2529-0.1484,0.3516
+              C23.252,16.9199,23.1348,16.9688,23,16.9688h-6v6c0,0.1357-0.0498,0.2529-0.1484,0.3516c-0.0996,0.0996-0.2168,0.1484-0.3516,0.1484
+              c-0.1357,0-0.2529-0.0488-0.3516-0.1484C16.0488,23.2217,16,23.1045,16,22.9688v-6h-6c-0.1357,0-0.2529-0.0488-0.3516-0.1484
+              C9.5493,16.7217,9.5,16.6045,9.5,16.4688c0-0.1348,0.0493-0.252,0.1484-0.3516C9.7471,16.0186,9.8643,15.9688,10,15.9688h6v-6
+              c0-0.1353,0.0488-0.2524,0.1484-0.3516c0.0986-0.0986,0.2158-0.1484,0.3516-0.1484c0.1348,0,0.252,0.0498,0.3516,0.1484
+              C16.9502,9.7163,17,9.8335,17,9.9688v6h6C23.1348,15.9688,23.252,16.0186,23.3516,16.1172z"
+      />
+      <path
```

Comment:
To create different variations of same svg, please follow the example in the teams.tsx svg file.

Reply:
@alinais Resolved